CHANGELOG.md

```diff
@@ -37,6 +37,7 @@ This project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.htm
 - The `Event` and `Link` struct types from the `go.opentelemetry.io/otel` package now include a `DroppedAttributeCount` field to record the number of attributes that were not recorded due to configured limits being reached. (#1771)
 - The Jaeger exporter now reports dropped attributes for a Span event in the exported log. (#1771)
 - Adds `k8s.node.name` and `k8s.node.uid` attribute keys to the `semconv` package. (#1789)
+- Adds `resource.NewEmptyResouce()` for creating a Resouce without builtin detectors (#1810)
```

Comment:
```suggestion
- Adds `resource.NewEmptyResource()` for creating a Resource without builtin detectors (#1810)
```

`NewEmptyResource` instead of `NewEmptyResouce`?